lib/ws_servers/api/handlers/on_safe_close.js

```diff
@@ -21,9 +21,7 @@ module.exports = async (server, session) => {
     }
   }
 
-  const r = await Promise.allSettled(tasks)
-
-  console.log(r)
+  await Promise.allSettled(tasks)
```

Comment:
may be we can return errors here (so that FE can notify users) if promise was threw any error, to let users know if some orders or strategies were not cancelled/not closed.